### DISCUSSION TITLE
Custom Template: can we swap/decorate BindingManager?

### DISCUSSION THREAD

**Discussion by rnice01** [wailsapp/wails#2216] [Q&A]
Hello,
I have a custom Elm template and would like to generate Elm types similar to how TypeScript definitions are created. I was digging around and it seems like there aren't any public interfaces for replacing the App's BindingManager. Is there a way to do this currently without modifying Wails itself?

Thanks!

**Comment by leaanthony** [comment]
Hi! That's a great idea. I think the bindings generator *should* be pluggable. I will create a ticket for this 👍 
EDIT: https://github.com/wailsapp/wails/issues/2217
